### PR TITLE
Add README.md with Hostnet sniff descriptions

### PR DESCRIPTION
**Pull request by FreddyOffenga**
Fixed issue #17 with @janlam7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
+## Rules Overview
```
Comment (nicoschoenmaker) [inline README.md:4]: Should have an initial line for introducing the package. That we add additional sniffs and fixes to squizlabs/php_codesniffer

File: README.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+class MyClass
+{
+    public abstract function getEntityManager();
+}
+```
+
+:+1:
+
+```php
+abstract class MyClass
```
Comment (nicoschoenmaker) [inline README.md]: We actually have a different sniff that obligates prefixing abstract classes with `Abstract`, so I would call good `AbstractMyClass`.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
+## Rules Overview
+
+### Abstract class must be prefixed with Abstract
+
+- class: [`Hostnet\Sniffs\Classes\AbstractClassMustBePrefixedWithAbstractSniff`](/src/Hostnet/Sniffs/Classes/AbstractClassMustBePrefixedWithAbstractSniff.php)
+- sniff: Hostnet.Classes.AbstractClassMustBePrefixedWithAbstract
```
Comment (hboomsma) [inline README.md]: may wrap the sniff name in back-ticks, so it will look like `this`

**Review by hboomsma — APPROVED** [review]
I see only small fixes, so I do approve...

**Review by nicoschoenmaker — APPROVED** [review]

**Comment by TomasVotruba** [comment]
So beautiful, thanks :heart:

**Pull request merged**